src/views/Collectibles/components/NftGiveawayModal.tsx

```diff
@@ -32,25 +29,12 @@ const showConfetti = () => {
   })
 }
 
-const getClaimableNft = (profile: Profile): Nft => {
-  if (!profile) {
-    return null
-  }
-
-  if (!profile.team) {
-    return null
-  }
-
-  const identifier = Object.keys(teamNftMap).find(
-    (mapNftIdentifier) => teamNftMap[mapNftIdentifier] === profile.team.id,
-  )
-  return nftList.find((nft) => nft.identifier === identifier)
+interface NftGiveawayModalProps extends InjectedModalProps {
+  nfts: Nft[]
 }
 
-const NftGiveawayModal: React.FC<InjectedModalProps> = ({ onDismiss }) => {
+const NftGiveawayModal: React.FC<NftGiveawayModalProps> = ({ onDismiss, nfts }) => {
```

Comment:
Modifying the GiveAway modal to receive an array of nfts and render their images. Any `can user claim nft` logic is to be handled within the `GlobalCheckClaimStatus` hook